src/dmifn.rs

```diff
@@ -589,15 +589,14 @@ pub fn dmi_processor_id(data: &SMBiosProcessorInformation<'_>) {
                     2 => {
                         println!(
                             "\tSignature: Family {}, Model {}, Stepping {}",
-                            (eax >> 8)
-                                & 0xF
+                            ((eax >> 8) & 0xF)
                                     + match ((eax >> 8) & 0xF) == 0xF {
-                                        true => (eax >> 20) & 0xFF,
+                                        true => ((eax >> 20) & 0xFF),
                                         false => 0,
                                     },
-                            (eax >> 4) & 0xF
+                            ((eax >> 4) & 0xF)
                                 | match ((eax >> 8) & 0xF) == 0xF {
-                                    true => (eax >> 12) & 0xF0,
+                                    true => ((eax >> 12) & 0xF0),
```

Comment:
Same issue as above: The parentheses here inside the true match arm have no effect (no other operators to resolve precedence with) and their presence may be confusing.